### PR TITLE
bump: kubebuilder 3.5.0 release

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description of the change:**

**Motivation for the change:**

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

**Review by everettraven — APPROVED** [review]
/lgtm

This looks good to me, but I think it would be a good idea to get another LGTM before merging just for verification.

**Review by rashmigottipati — APPROVED** [review]
/lgtm

**Review by laxmikantbpandhare — APPROVED** [review]
/lgtm

Just one quick question about the below sanity error

```
internal/plugins/ansible/v1/init.go:19: File is not `goimports`-ed (goimports)
	sdkpluginutil "github.com/operator-framework/operator-sdk/internal/plugins/util"
```

It looks like we have to make it **local**. Is it necessary or can we avoid it?

File: internal/plugins/ansible/v1/init.go
Comment (camilamacedo86) [inline internal/plugins/ansible/v1/init.go:19]:
```suggestion
```

**Pull request merged**
